python/ray/_raylet.pyx

```diff
@@ -1726,7 +1726,6 @@ cdef class CoreWorker:
             CObjectID c_object_id = object_ref.native()
             CAddress c_owner_address = CAddress()
             c_string serialized_object_status
-        CCoreWorkerProcess.GetCoreWorker().PromoteObjectToPlasma(c_object_id)
```

Comment:
Aren't there other cases where we promote, like during arg passing?

Reply:
I think it's actually only used here. Arg passing also goes through this serialization codepath.